can_tools/scrapers/base.py

```diff
@@ -617,6 +617,9 @@ def find_unknown_variable_id(self, engine: Engine, df: pd.DataFrame):
 
     def find_unknown_location_id(self, engine: Engine, df: pd.DataFrame):
         locs = pd.read_sql("select * from locations", engine)
+        if "location" in df.columns:
```

Comment:
Not related to the scraper, but this function helps find any missing/unknown locations in the scraper output. Previously it only worked on scrapers that track locations by county name, so this adds the ability to find unknown fips codes, too.